src/pages/components/range.mdx

```diff
@@ -1,10 +1,116 @@
 ---
 title: Range
 description: >
-  Description will be here
+  A Range lets users choose a single number between two values by using a slider.
+package: '@zendeskgarden/react-forms'
+components:
+  - forms/src/elements/common/Field.tsx
+  - forms/src/elements/common/Hint.tsx
+  - forms/src/elements/common/Label.tsx
+  - forms/src/elements/common/Message.tsx
+  - forms/src/elements/Range.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To choose a number within a range with minimum and maximum values</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          If the choices aren’t in a continuous range, use a [Menu](/components/menu) instead
+        </Markdown>
+      </li>
+      <li>
+        <Markdown>
+          To compare between several options, use a [Radio](/components/radio) instead
+        </Markdown>
+      </li>
+      <li>
+        <Markdown>
+          To define a range of values, use a [Multi-thumb range](/components/multi-thumb-range)
+          instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Range component.
+
+import Default from '../../examples/forms/range/RangeDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/forms/range/RangeDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Disabled
+
+A disabled Range prevents user interaction. It doesn’t appear in the tab order,
```

Comment:
A disabled Range is not interactive, and not perceived by a screen reader. See Disabled UI pattern guidelines for best practices.